packages/lablgtk3-gtkspell3/lablgtk3-gtkspell3.3.0.beta6/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+
+synopsis: "OCaml interface to GTK+3"
+description: """
+OCaml interface to GTK+3, gtkspell library
+
+See http://lablgtk.forge.ocamlcore.org/ for more information.
+
+"""
+
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "https://github.com/garrigue/lablgtk"
+bug-reports: "https://github.com/garrigue/lablgtk/issues"
+dev-repo: "git+https://github.com/garrigue/lablgtk.git"
+doc: "https://garrigue.github.io/lablgtk/lablgtk3-gtkspell3"
+license: "LGPL with linking exception"
+
+depends: [
+  "ocaml"                {         >= "4.05.0"    }
+  "dune"                 { build & >= "1.4.0"
+                                 & != "1.7.0"
+                                 & != "1.7.1"     } # Due to dune/dune#1833
+  "dune"                 { build & >= "1.4.0"     }
```

Comment:
```suggestion
```
This dependency is present twice

Reply:
Looks like @ejgallego to remove it in this file. Fixed as suggested.

Reply:
Yup, likely some merge glitch, sorry!

p.s. @garrigue a pitfall of the fast-forward merge approach, my local git loses tracks of which commits were applied to the master branch and I have to be more careful with the rebases.

Reply:
Yup, likely some merge glitch, sorry!

p.s. @garrigue a pitfall of the fast-forward merge approach, my local git loses tracks of which commits were applied to the master branch and I have to be more careful with the rebases.